### PR TITLE
app-admin/yadm: Bump version to 1.07

### PR DESCRIPTION
**Pull request by esphen**
Adds new dependency `dev-tcltk/expect` for tests

Package-Manager: portage-2.3.3

### CODE REVIEW

**Comment by kensington** [comment]
Any reason for changing ``emake`` to ``make``?

File: app-admin/yadm/yadm-1.07.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+RDEPEND="dev-vcs/git
+	app-crypt/gnupg"
+
+src_compile() {
+	make yadm.md
+}
+
+src_test() {
+	# 109_accept_encryption tests are interactive, thus fail. Skip them
+	bats $(find test/ -name '*.bats' -and -not -name '109_accept_encryption.bats') \
```
Comment (SoapGentoo) [inline app-admin/yadm/yadm-1.07.ebuild]: please use funky while read loop (with `-print0` here) here: https://devmanual.gentoo.org/tools-reference/find/

**Comment by esphen** [comment]
@kensington No. Not sure how that got in there
@SoapGentoo Okay. Can I ask why?

I amended the commit with the requested changes

**Comment by kensington** [comment]
Thanks! Picked as bbee50cc7789c56f81c6909f4a0ff8da1f0c6a01.

I reordered the dependencies slightly and squashed it into the commit - we generally prefer to keep them sorted.

**Pull request closed without merging**